prediction_market_agent_tooling/markets/seer/seer.py

```
@@ -407,19 +407,56 @@ def from_data_model_with_subgraph(
         price_manager = PriceManager(seer_market=model, seer_subgraph=seer_subgraph)
```

Ruff (0.12.2)

Local variable `price_manager` is assigned to but never used

Remove assignment to unused variable `price_manager`

(F841)